photon-lib/py/photonlibpy/photonCamera.py

```diff
@@ -165,11 +165,31 @@ def _versionCheck(self) -> None:
 
         versionString = self.versionEntry.get(defaultValue="")
         if len(versionString) > 0 and versionString != PHOTONVISION_VERSION:
-            wpilib.reportWarning(
-                "Photon version "
-                + PHOTONVISION_VERSION
-                + " does not match coprocessor version "
-                + versionString
-                + f"! Please install photonlibpy version {PHOTONLIB_VERSION}",
-                True,
-            )
+            # Verified version mismatch
+
+            bfw = ""
```

Comment:
Use a multiline string instead. Those use triple quotation marks.

Reply:
Aren't those Java-17 specific? We still target java 11 in photon-lib